model/src/main/java/org/cbioportal/genome_nexus/model/PfamDomain.java

```diff
@@ -1,114 +1,44 @@
 package org.cbioportal.genome_nexus.model;
 
-import com.univocity.parsers.annotations.Parsed;
-import com.univocity.parsers.annotations.Trim;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
 
 /**
  * @author Selcuk Onur Sumer
  */
+@Document(collection="pfam.domain")
```

Comment:
Instead of a reading a file at runtime, now we are using a mongo db collection for PFAM